contracts/src/token/erc20/extensions/burnable.rs

```diff
@@ -3,7 +3,7 @@
 use alloy_primitives::{Address, U256};
 use stylus_sdk::msg;
 
-use crate::token::erc20::{Erc20, Error};
+use crate::token::erc20::{self, Erc20, Error};
```

Comment:
```suggestion
use crate::token::erc20::{self, Erc20};
```
Let's remove this `Error` import, and [reference it](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/7c41ab349143a4c6667d915ab505ad123973de90/contracts/src/token/erc20/extensions/burnable.rs#L63) like `type Error = erc20::Error` - makes it clearer that we're reusing ERC-20 errors